src/app/settings/views/Storage/StorageForm/StorageForm.test.tsx

```diff
@@ -26,7 +26,7 @@ describe("StorageForm", () => {
         items: [
           {
             name: ConfigNames.DEFAULT_STORAGE_LAYOUT,
-            value: "bcache",
+            value: "flat",
```

Comment:
I'm assuming you made this change in order for form to even allow submission in the test below (as we require values to be different)?

Reply:
Precisely!